coralproject/talk
# Added --force to rebuild native modules

## What does this PR do?

Adds a `--force` flag to the yarn install process so that when installing dependancies, we ensure that native modules are rebuilt.

## How do I test this PR?

1. Build images locally: `./scripts/docker.sh`
2. Create a new folder (elsewhere on your harddrive) and create a Dockerfile in it with the following content:

```Dockerfile
FROM coralproject/talk:latest-onbuild
```

3. Build the docker image: `docker build -t coralproject/talk:pr-test .`
4. Run the image `docker run --rm -ti coralproject/talk:pr-test bash`
5. Run the following `node models/user.js`

There should be an error (specifically about not connecting to the database), but you should no longer see an error related to `bcrypt` not being installed.

This issue is related to an upstream issue with yarn: https://github.com/yarnpkg/yarn/issues/756
